modules/install-osp-dualstack.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+:_content-type: CONCEPT
+[id="install-osp-dualstack_{context}"]
+= Optional: Configuring a cluster with dual-stack networking
+
+:FeatureName: Dual-stack configuration for OpenStack
+include::snippets/technology-preview.adoc[]
+
+You can create a dual-stack cluster on {rh-openstack}. However, this is enabled only if you are using an {rh-openstack} network with IPv4 and IPv6 subnets. To configure a dual-stack cluster, you must complete the following tasks:
+
+* Creating a network with IPv4 and IPv6 subnets. The available address modes for `ipv6-ra-mode` and `ipv6-address-mode` fields are: `stateful`, `stateless` and `slaac`.
+
+** The dualstack network MTU must accommodate both the minimum MTU for IPv6, which is 1280, and the OVN-Kubernetes encapsulation overhead, which is 100.
+
+** DHCP must be enabled on the subnets.
+
+* Creating API and Ingress VIPs ports.
+
+* Adding the IPv6 subnet to a neutron router to enable router advertisements.
```

Comment:
```suggestion
* To enable external connectivity through IPv6 and router advertisements, the user needs to add the dual-stack Network as external gateway in the Router.
```

Reply:
Tweaking this a bit to adhere to writing principles